src/main/webapp/data/developers.json

```diff
@@ -1601,21 +1613,41 @@
       "username": "darrenwee",
       "startPeriod": "Jan 2018"
     },
+    {
+      "name": "Jacob Li",
+      "username": "jacoblipech",
+      "image": "images/teammembers/jacob.png",
+      "startPeriod": "Aug 2018"
```

Comment:
They started at May 18, no?

Reply:
Should it be the time they get promoted?

Reply:
If you look around, it's not the case for everyone else (you and me included).